dislib/classification/rf/data.py

```diff
@@ -0,0 +1,199 @@
+from numpy.lib import format
+import numpy as np
+import tempfile
+
+from pycompss.api.parameter import FILE_IN, FILE_INOUT
+from pycompss.api.task import task
+
+from dislib.data import Dataset
+
+
+@task(labels_path=FILE_IN, returns=3)
+def get_labels(labels_path):
+    y = np.genfromtxt(labels_path, dtype=None, encoding='utf-8')
+    def_intp = np.intp  # Save default np.intp
+    np.intp = np.int8  # Modify np.intp, so that codes.dtype=int8
+    categories, codes = np.unique(y, return_inverse=True)
```

Comment:
Can't this be done by passing dtype=np.int8 to np.genfromtxt?

Reply:
The mangling with the numpy dtypes has been removed (it wasn't related to np.genfromtxt).